app/helpers/auth.rb

```diff
@@ -0,0 +1,39 @@
+ helpers do
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by_id(session[:user_id])
+    end
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
Best to follow Ruby convention for coercing an object into its equivalent boolean value:
```ruby
def logged_in?
  !!current_user
end
````